file_writer_control/JobHandler.py

```diff
@@ -91,10 +91,8 @@ def abort_write_job(self) -> Union[CommandHandler, None]:
         :return: A CommandHandler instance that can be used to monitor the outcome of the attempt to set a new stop time.
         """
         current_status = self.worker_finder.get_job_status(self._job_id)
-        if current_status is None:
-            return None
         return self.worker_finder.try_send_abort(
```

Comment:
Is it still possible for the function to return None? We might need to update the return type in line 79.